### ISSUE TITLE
Duplicate Lines in Participations -> Submissions View

### ISSUE THREAD

**Issue by egekurt123** [ls1intum/Artemis#8888]
### Describe the bug

"In the submissions view, the lines in categories 'Results', 'Assessment Type', and 'Completion Date' appear duplicated, as seen in the attached screenshot."

### To Reproduce

Steps to Reproduce
1. Navigate to the submissions -> participations view.
2. Observe the duplicated lines for each submission.

### Expected behavior

Each category should be displayed only once in the participations view.

### Screenshots

![New Issue](https://github.com/ls1intum/Artemis/assets/69898283/22fde6b3-b1ee-4f89-a7d1-2caf4d91139a)

### Which version of Artemis are you seeing the problem on?

7.3.1

### What browsers are you seeing the problem on?

Chrome, Safari, Microsoft Edge, Firefox

**Comment by Strohgelaender** [comment]
This is a correct behaviour, a submission can have multiple results that then get displayed like this.
